src/include/homestore/replication/repl_decls.h

```diff
@@ -78,6 +78,7 @@ struct peer_info {
     uint64_t replication_idx_;
     // The elapsed time since the last successful response from this peer, set to 0 on leader
     uint64_t last_succ_resp_us_;
+    bool is_new_joiner{false};
```

Comment:
Lets do it in the grpc spec clearly state which field is trustable.

Now ever member in PG report status,  the replication_idx only valid on leader (other replica are all zero) and seems like the is_new_joiner only valid for local node?